main.tf

```
@@ -55,13 +59,41 @@ resource "aws_vpn_connection" "default" {
   tunnel2_phase1_integrity_algorithms  = var.vpn_connection_tunnel2_phase1_integrity_algorithms
   tunnel2_phase2_integrity_algorithms  = var.vpn_connection_tunnel2_phase2_integrity_algorithms
 
+  tunnel1_log_options {
+    cloudwatch_log_options {
+      log_enabled = var.vpn_tunnel_logging
+      log_group_arn = aws_cloudwatch_log_group.tunnel1[0].arn
```

If `vpn_tunnel_logging` is `false`, I believe this will fail because there will not be a resource at position `0`. Let's update to the following: 

```suggestion
      log_group_arn = join("", aws_cloudwatch_log_group.tunnel1.*..arn)
```

Same for below.